packages/dart_frog_gen/lib/src/build_route_configuration.dart

```diff
@@ -105,11 +105,12 @@ List<RouteFile> _getRouteFilesForDynamicDirectories(
   String prefix = '',
 }) {
   final files = <RouteFile>[];
+  final thisIsDynamic = directory.isDynamicRoute;
```

Comment:
```suggestion
  final isDynamic = directory.isDynamicRoute;
```

Calling it this feels strange as it doesn't refer to the current class but to a value out of it

Reply:
Agreed